### ISSUE TITLE
🖐️ Change Node version that Dagger is using

### ISSUE THREAD

**Issue by noah-troncoso** [dagger/dagger#7855]
### What happened? What did you expect to happen?

Dagger is currently using Node 21, which is breaking my pipeline because certain packages only support LTS versions of Node. This should get resolved by https://github.com/dagger/dagger/issues/7433, whenever that releases, but is there a workaround in the mean time? My CI/CD pipelines are failing due to this.

**Comment by levlaz** [comment]
Hey @noah-troncoso was this working and broke, or was it never working due to this issue? 

The fix has already been merged, so you should be able to run dagger from main if this is urgent, I think that may be the best workaround for now until this is released  https://github.com/dagger/dagger/issues/7855

I am checking to see if we are planning to include this in our upcoming release this week, ill follow up here.

**Comment by levlaz** [comment]
@noah-troncoso confirming that this will be shipped in the next release of Dagger 0.12 which *should* be out this week.

**Comment by noah-troncoso** [comment]
@levlaz thanks for the response. Yeah, on Monday things were working fine. On Tuesday, I updated some README content and started getting this error. I always specify Dagger version 0.11.8, and the Github Action logs show this to be the case. The working runs from Monday don't seem to print the Node version being used, so I can't actually confirm that it changed from one day to the next.

Here is the exact error:

![Screenshot 2024-07-10 at 8 35 21 AM](https://github.com/dagger/dagger/assets/123982978/f6f0e598-fdf0-43e8-ae3a-6f093616ad2e)

This is literally the only file I changed:

![Screenshot 2024-07-10 at 8 36 16 AM](https://github.com/dagger/dagger/assets/123982978/6c2c8a64-6890-4fc9-a0e4-e629b146fc6f)

Thanks for the work around. If the release will be this week or at least next week, I can probably wait.
